style.css

```diff
@@ -1,3 +1,122 @@
-#root {
-  color: red;
+/*  
+=============== 
+Global Styles
+===============
+*/
+
+*,
+body {
+  background-color: whitesmoke;
+  line-height: 1.5;
+  font-size: 1rem;
+}
+
+a {
+  text-decoration: none;
+}
+img:not(.logo) {
+  width: 100%;
+}
+
+*/ h1,
+h2,
+h3,
+h4 {
+  text-transform: capitalize;
+}
+
+h1 {
+  font-size: 4rem;
+}
+h2 {
+  font-size: 2rem;
+}
+h3 {
+  font-size: 1.5rem;
+}
+h4 {
+  font-size: 0.875rem;
+}
+p {
+  margin-bottom: 1.25rem;
+  color: rgb(37, 37, 37);
+}
+.cardHeader {
+  display: flex;
+  justify-content: space-around;
+}
+#selectedEpisode {
+  min-width: 200px;
+  margin-right: 20px;
+}
+
+@media screen and (min-width: 640px) {
+  h1 {
+    font-size: 3rem;
+  }
+  h2 {
+    font-size: 1.5rem;
+  }
+  h3 {
+    font-size: 1.25rem;
+  }
+  h4 {
+    font-size: 1rem;
+  }
+  body {
+    font-size: 1rem;
+  }
+  h1,
+  h2,
+  h3,
+  h4 {
+    line-height: 1.5;
+  }
+  #page-header {
+    width: 90vw;
+    padding: 10px;
+    margin: 0 auto;
+    display: flex;
+    justify-content: flex-start;
+    /* border: 2px solid yellowgreen; */
+  }
+  #root {
+    color: #19272e;
+    width: 90vw;
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    grid-gap: 20px;
+  }
+}
+
```

Comment:
Great to see you trying to implement responsive design too. My only comment would be as to why you have breakpoints for a small screen and what seems to be a large(r) screen, but no medium?